src/plugins/timeConductor/Conductor.vue

```diff
@@ -97,7 +104,7 @@
                     autocorrect="off"
                     spellcheck="false"
                     :disabled="!isFixed"
-                    @change="validateAllBounds('endDate'); submitForm()"
+                    @change="validateAllBounds('endDate'); submitForm ()"
```

Comment:
Looks like some extra whitespace has snuck in here

Reply:
Fixed.